pkg/scheduler/objects/queue_test.go

```diff
@@ -2015,6 +2016,7 @@ func TestApplyConf(t *testing.T) {
 	assert.Assert(t, leaf.template == nil)
 	assert.Assert(t, leaf.maxResource != nil)
 	assert.Assert(t, leaf.guaranteedResource != nil)
+	assert.Assert(t, leaf.maxRunningApps == 100)
```

Comment:
Would `assert.Equal()` be better in this case? It can provide more detail when failed.

Reply:
Thank you @ryankert01 for review, addressed in latest PR.